lib/citgm.js

```diff
@@ -4,6 +4,12 @@
 var util = require('util');
 var EventEmitter = require('events');
 
+// This is a shim to allow for v0.10 to work
+// TODO remove when we no longer support v0.10
+if (EventEmitter.EventEmitter) {
```

Comment:
thanks for the heads up. fixed in https://github.com/TheAlphaNerd/citgm/commit/51dac61db90d7949489bb13085bc9d747f5ed176